app/controllers/suppliers_controller.rb

```diff
@@ -57,6 +59,16 @@ def supplier_with_top_three
     @supplier_with_top_three ||= Supplier.fetch_with_top_three(permitted_params[:id])
   end
 
+  def set_quarter_date
+    # passes in contentful id for quarter dates content
+    @quarter_date = QuarterDate.fetch_quarter_dates_content("3OdGRuiq5a99kVxYpQQklF")
```

Comment:
Do you think it's likely these ids will change for any reason?

If you miss the readability of the names, you could use constants eg

```
class SuppliersController < ApplicationController
  QUARTER_DATE_CONTENT_ID = "3OdGRuiq5a99kVxYpQQklF"
  NEXT_QUARTER_DATE_CONTENT_ID = "230w36VUwaerIjibJYFoSG"
 ...
```

Reply:
I hope they won't change, but can't guarantee that no-one would ever delete them.

Good idea re using constants - thanks!